doc/R.nvim.txt

```diff
@@ -924,10 +924,11 @@ Note: The options of this section are ignored on macOS, where the command
 `open` is called to run the default application used to run shell scripts.
 
 R will run in Neovim's built-in terminal emulator, unless you set the value of
-`external_term`, which can be a boolean value, the name of the terminal
-emulator or the complete command to run the terminal emulator. On Windows,
-only boolean values are valid; if `true` R.nvim will run `Rgui.exe`, and not
-really a terminal emulator.
+`external_term`, which can be `""` (use Neovim's built-in terminal),
```

Comment:
Should not be default? or both "default" and "" are valid for the same thing?

Reply:
Do you have any suggestion of how to improve the documentation wording?

Reply:
I think this is good. I wrote this comment without reading further. But I think you got it clear.